src/controllers/NotificationController.ts

```diff
@@ -26,6 +26,83 @@ class NotificationController {
       return res.status(500).json({ message: 'Internal server error' })
     }
   }
+  /**
+   * Mark a notification as read or unread
+   * @param {Request} req - Express request object
+   * @param {Response} res - Express response object
+   * @returns {Promise<Response>} Promise that resolves to an Express response
+   */
+  static async changeNotificationStatus(
+    req: Request,
+    res: Response,
+  ): Promise<Response> {
+    try {
+      const { notificationId } = req.params;
+      const { isRead } = req.body;
+
+      const notification = await Notification.findOne({
+        where: { id: notificationId },
+      });
+
```

Comment:
Maybe you can check if the user is going to update their notification, which means a user cannot mark a notification as read if it does not belong to them.

Try to check on that.